README.md

```diff
@@ -1,4 +1,43 @@
-# laa-apply-for-legalaid-api
-This api will povide various services to create a legal aid application
 
-Test
+#LAA apply for legal aid
+
+
+This is the service api for persisting case related information to the back end database and
+may well be used to fire requests to other services.
+
+* Ruby version 
+    * Ruby version 2.5.1
+    * Rails 5
+
+
+* System dependencies
+    * none
+
+* Configuration
+    * ```bundle install```
```

Comment:
The purpose of that bundle install i think is to update the gemfile lock and make sure the developer has the required gems locally.

Reply:
So, after conversations offline, this happens because `rbenv` sets a non-default `BUNDLE_PATH`. I'm using `chruby` which doesn't. Everyone else is using `rbenv` so we can ignore this for now.